packages/zone/src/virtual.js

```diff
@@ -100,13 +66,14 @@ export const makeVirtualZone = (baseLabel = 'virtualZone') => {
     baseLabel,
     detachedVirtualStores,
   );
-  return Far('VirtualZone', {
-    exo: wrapProvider(defineVirtualExo, keys.exo),
+  const subZoneProvider = (label, _options) =>
```

Comment:
```suggestion

  /**
   * @param {string} label
   * @param {any} _options
   */
  const provideSubZone = (label, _options) =>
```

Reply:
Param types added. Renaming not for stated reasons.